### ISSUE TITLE
Autobalance Balance-Intervall Setting

### ISSUE THREAD

**Issue by RRonGit** [shining-man/bsc_fw#150]
Was ich noch gut finden würde, wenn man die Einstellung "Balance-Intervall" an Wochentagen setzten könnte.
z.B. jeden SA oder jeden zweiten SA u.s.w.

diese Einstellung hier:

![image](https://github.com/user-attachments/assets/79acd71e-5258-4d18-9233-6c2947c6e9ce)

**Comment by holle75** [comment]
recht spannende Idee. Mir ist auch gerade aufgefallen, dass dank Neustart ich dann heute wieder balancen werde. Wenn es eine Logik "Balance-Intervall x Tage, aber nicht vor Wochentag(en) y" gäbe ... könnte man sich das schön auf Tage mit eher größerer Abwesenheit zB legen.

**Comment by D3R-ST3FAN** [comment]
Man kann Balcing ja auch via Trigger anstoßen.
Ich hab was ähnliches mit einem Trigger-Schedule gemacht

**Comment by holle75** [comment]
jo, auch eine idee. Wenn du Lust hast, zeig mal deinen Code/Logik, dann kann man das schnell nachbauen und muss das Rad nicht neu erfinden. Egal in welcher Automation.

**Comment by D3R-ST3FAN** [comment]
Hier mal schnell aus dem [Simulator](https://doc.battery-controller.de/demo/app.html)
<img width="577" height="345" alt="Image" src="https://github.com/user-attachments/assets/cbe7c927-a747-48e8-8580-4f5b275c2b5d" />

<img width="589" height="539" alt="Image" src="https://github.com/user-attachments/assets/50e432ba-46ff-4be3-bf0f-962819131131" />

Nix besonderes.
Den Intervall höher stellen als den "Abstand" im Scheduler.

**Comment by holle75** [comment]
Ah ok, ich bin free unterwegs. Da gibts den Trigger nicht.

**Comment by D3R-ST3FAN** [comment]
Vielleicht könnte man die Intervallmöglichkeiten noch verlängern und/oder einen Sonderintervall 0 einführen, wo dan Balancing NUR über trigger ausgeführt wird.
